### DISCUSSION TITLE
[Kavita] Idea: Display WebLinks from ComicInfo.xml (Manga)

### DISCUSSION THREAD

**Discussion by Fesaa** [Kareadita/Kavita#3306] [Ideas]
### Idea Description

I'd love for Kavita to read and display WebLinks saved in the ComicInfo.xml file

From what Joe said in the original issue, this would only be for Manga's. 

Extra info:
Original Issue: https://github.com/Kareadita/Kavita/issues/3254
I searched through the FRs, but didn't find any. So posting this, as I'd really like it. Sorry if I missed it, please do link so I can upvote that one then 😓 

### Idea Category

Feature Enhancement

### Duration of Using Kavita

I'm not sure

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by majora2007** [comment]
Guess whomever I spoke to in discord never created the FR.
